span.go

```diff
@@ -121,6 +124,18 @@ type LogData struct {
 	Payload interface{}
 }
 
+// FinishOptions allows Span.FinishWithOptions callers to override the finish
+// timestamp and provide log data via a bulk interface.
+type FinishOptions struct {
+	// FinishTime overrides the Span's finish time, or implicitly becomes
+	// time.Now() if FinishTime.IsZero().
+	FinishTime time.Time
+
+	// BulkLogData allows the caller to specify the contents of many Log()
+	// calls with a single slice. May be nil.
+	BulkLogData []LogData
```

Comment:
what are the semantics for the Span's start/finish time if conflicting LogData is inserted?